src/popups/ZweitausbildungRoutesPopup/ZweitausbildungRoutesPopup.js

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Feature from 'ol/Feature';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+
+import './ZweitausbildungRoutesPopup.scss';
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+class ZweitausbildungRoutesPopup extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    // Sorted unique features by bezeichnung
+    const singleFeatures = props.feature
+      .get('features')
+      .filter((feat, pos, arr) => {
+        return (
+          arr
+            .map(f => f.get('bezeichnung'))
+            .indexOf(feat.get('bezeichnung')) === pos
+        );
+      })
+      .sort((a, b) => (a.get('bezeichnung') > b.get('bezeichnung') ? 1 : -1));
+
+    this.state = {
+      features: singleFeatures,
+    };
+  }
+
+  componentDidMount() {
+    const { features } = this.state;
+
+    // Highlight the first feature
+    this.highlight(features[0]);
+  }
+
+  componentWillUnmount() {
+    const { feature } = this.props;
+    const highlightFeatures = feature.get('highlightFeatures');
+
+    // Update the features in the map
+    for (let i = 0; i < highlightFeatures.length; i += 1) {
+      highlightFeatures[i].set('highlight', false);
+      highlightFeatures[i].changed();
+    }
+  }
+
+  highlight(singleFeature) {
+    const { feature } = this.props;
+    const { features } = this.state;
+    const highlightFeatures = feature.get('highlightFeatures');
+
+    // Update the features in the popup
+    const singleFeatures = features;
+    for (let j = 0; j < singleFeatures.length; j += 1) {
+      singleFeatures[j].set('highlight', singleFeatures[j] === singleFeature);
+    }
+
+    // Update the features in the map
+    for (let i = 0; i < highlightFeatures.length; i += 1) {
+      highlightFeatures[i].set(
+        'highlight',
+        highlightFeatures[i].get('bezeichnung') ===
+          singleFeature.get('bezeichnung'),
+      );
+      highlightFeatures[i].changed();
+    }
+
+    this.setState({ features: singleFeatures });
+
+    this.forceUpdate();
```

Comment:
@oterral Do you know why the re-rendering only works with `this.forceUpdate()`?

Reply:
The set state doesn;t set a new state but the same value?

Reply:
The setState doesn't set a new state but the same value?

Reply:
create a new array should avoid the use of forceUpdate:
```
this.setState({ features: [...singleFeatures] });
```